services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/email/EmailSenderService.kt

```diff
@@ -158,4 +148,16 @@ class EmailSenderService(
 
     private fun parseTemplate(path: String, contextObject: Any): String =
         Template.parse(path.getResource().openStream()).processToString(contextObject)
+
+    companion object {
+        fun ApplicationType.translations() =
```

Comment:
Why does this extension need to be in a companion object?

Reply:
An extension function in a class cannot be imported from outside that class. This makes sense since there's no way to tell which instance of the class should be used for the call.